packages/integration-tests/bin/cli.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const pckg = require('../package.json');
+const testRunner = require('../index');
+const program = require('commander');
+
+program.version(pckg.version);
+
+program
+  .usage('TYPE COMMAND [options]')
+  .option('-s, --stack-name <stackName>', 'AWS Cloud Formation stack name', null)
```

Comment:
Since this command will fail if any of these options are not set, that should be detected and handled.  Right now there is nothing checking to see if these options (all of which are necessary) are set.

Reply:
Done